pbdf/Issues/surfnet-2/description.xml

```diff
@@ -62,28 +62,28 @@
 				<nl>Type</nl>
 			</Name>
 			<Description>
-				<en>Your position at your institute</en>
-				<nl>Uw positie bij uw instantie</nl>
+				<en>Your role at your institution</en>
+				<nl>Je rol bij je instelling</nl>
 			</Description>
 		</Attribute>
 		<Attribute id="id" optional="true">
 			<Name>
-				<en>User ID</en>
-				<nl>Gebruiker ID</nl>
+				<en>Registration number</en>
+				<nl>Registratie nummer</nl>
```

Comment:
`Registratienummer` is a single word.

Reply:
Thanks @confiks! I will change this together with the rest (once everything has been discussed).

Reply:
Right.